internal/runtimehandlerhooks/utils_linux.go

```diff
@@ -228,3 +243,44 @@ func fileExists(filename string) bool {
 
 	return !info.IsDir()
 }
+
+// calculateCPUSizeFromMask return the number of total cpus
+// given a CPU mask
+// this is useful for finding the complement of a mask.
+func calculateCPUSizeFromMask(mask string) int {
+	noCommaMask := strings.ReplaceAll(mask, ",", "")
+	maskLen := len(noCommaMask)
+	// if mask is odd, round up
+	if maskLen%2 == 1 {
```

Comment:
Why? Is even number of cores (no-HT case) a reliable assumption?

Reply:
fair enough

Reply:
No need, just refreshed my memory about https://bugzilla.redhat.com/show_bug.cgi?id=2181546#c11 and rounding to the nearest byte boundary is fine. Just change the comment a bit to say something like:

"Make the hexadecimal mask full byte aligned in case the input ends with a half-byte."

Reply:
Discussing offline, according to https://bugzilla.redhat.com/show_bug.cgi?id=2181546#c11 it's fine